modules/workerpools/variables-cluster.tf

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022, 2023 Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+variable "apiserver_private_host" {
+  default     = ""
+  description = "Cluster private apiserver IP address. Resolved automatically when OKE cluster is found using cluster_id."
+  type        = string
+}
+
+variable "apiserver_public_host" {
+  default     = ""
+  description = "Cluster public apiserver IP address, when enabled. Resolved automatically when OKE cluster is found using cluster_id."
+  type        = string
+}
+
+variable "cluster_dns" {
+  default     = "10.96.5.5"
```

Comment:
Why is this value hardcoded?

Reply:
This is the default value within [KubernetesNetworkConfig#servicesCidr](https://docs.oracle.com/en-us/iaas/api/#/en/containerengine/20180222/datatypes/KubernetesNetworkConfig) used in most cases. Adding clarification to the description.